### PR TITLE
Save project

### PR DESCRIPTION
**Pull request by CyriacInfo**

### CODE REVIEW

File: src/interfaces/InputType.ts [hunk 1]
```diff
@@ -168,3 +168,22 @@ export class iCommentAnswer {
   @Field({ nullable: true })
   answer_date: Date;
 }
+
+@InputType()
+@ObjectType()
+export class iFilesWithCode {
+  @Field({ nullable: true })
+  id: number;
```
Comment (Chap-fr) [inline src/interfaces/InputType.ts:176]: no need ID in inputType (auto generated)

File: src/resolvers/fileResolver.ts [hunk 2]
```diff
@@ -127,40 +132,110 @@ export class FileResolver {
...
-      contentData
-    );
-    const result = {
-      success: true,
-    };
-    return JSON.stringify(result);
+    try {
+      // de l'id du fichier pour verifier si le fichier existe
+      // Verifier si le fichier appartient bien à l'utilisateur
+      console.log("*********************************",projectId, fileId, contentData)
```
Comment (Chap-fr) [inline src/resolvers/fileResolver.ts:211]: remove console log

File: src/tools/fileManager.ts [hunk 3]
```diff
@@ -103,17 +105,47 @@ export const fileManager = {
...
 
   updateContentData: async (
-    project: Project,
-    id_storage_file: string,
+    projectPath: string,
+    filepath: string,
     contentData: string
   ) => {
     try {
+      console.log("-----------------------------------",projectPath, filepath, contentData)
```
Comment (Chap-fr) [inline src/tools/fileManager.ts:113]: remove console log

File: src/resolvers/fileResolver.ts [hunk 4]
```diff
@@ -127,40 +132,110 @@ export class FileResolver {
...
+      const [project] = (await projectService.getById(
+        projId
+      )) as unknown as ReqProject[];
+
+      // const projectShare = await projectShareService.getUserCanEdit(projId);
+      // const thisUserCanEdit = projectShare.filter(
+      //   (share) => share.userId === ctx.id
+      // );
+
+      // if (project[0].userId.id !== ctx.id && thisUserCanEdit.length === 0)
```
Comment (Chap-fr) [inline src/resolvers/fileResolver.ts:179]: remove commented code

**Review by Chap-fr — CHANGES_REQUESTED** [review]

**Review by gnos28 — APPROVED** [review]

**Review by Chap-fr — APPROVED** [review]

**Pull request merged**